lxd/instance_backup.go

```diff
@@ -528,9 +528,11 @@ func instanceBackupPost(d *Daemon, r *http.Request) response.Response {
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["instances"] = []string{name}
-	resources["containers"] = resources["instances"]
+	resources := map[string][]api.URL{}
+	resources["instances"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", name, "backups", req.Name)}
```

Comment:
This URL isnt correct.

> In general you should ensure that the URLs stored inside a particular resource category represent the same category, for example the "instances" category should only contain URLs to instances, not backups or snapshots.

https://github.com/lxc/lxd/pull/11795#issuecomment-1589094231